<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>368</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>649</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>649</width>
    <height>378</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Brandcrowd</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/resources.qrc">
    <normaloff>:/images/images/logo.png</normaloff>:/images/images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Preview {
	border:1px solid #fafafa;
	border-radius:2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Sesstings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="BusinessName">
         <property name="placeholderText">
          <string>your name</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Keyword :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="KeyWord">
           <property name="placeholderText">
            <string>key word</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Logo Style :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="LogoStyle">
           <item>
            <property name="text">
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Icon &amp; Text</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Text-only</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Font Style :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="FontStyle">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Traditional(Serif)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bold</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vintage</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modern(Sans)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Funky</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Handwritten</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Color :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="Color">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Grayscale</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Purple</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pink</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Orange</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yellow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnCreateImage">
         <property name="text">
          <string>Create Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="Preview">
         <property name="minimumSize">
          <size>
           <width>278</width>
           <height>222</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>278</width>
           <height>222</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="back">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNext">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="next">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>649</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset theme="qt">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="filesave">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save Current Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
